vsts/scripts/tagRunTimeImagesForRelease.sh

```diff
@@ -7,7 +7,7 @@
 set -o pipefail
 
 outFileMCR="$BUILD_ARTIFACTSTAGINGDIRECTORY/drop/images/runtime-images-mcr.txt"
-outFileDocker="$BUILD_ARTIFACTSTAGINGDIRECTORY/drop/images/runtime-images-mcr.txt"
+outFileDocker="$BUILD_ARTIFACTSTAGINGDIRECTORY/drop/images/runtime-images-dockerhub.txt"
```

Comment:
isn't it about time we stopped releasing to DockerHub? :)

Reply:
i will be happy to scratch this out .. one less step to fail :P ..